t/coretests.pm

```diff
@@ -429,22 +435,26 @@ EOF
     }
 
 SKIP: {
-    skip "Cannot test \"use parent $CLASS\"  when require is used", 3
+    skip "Cannot test \"use base $CLASS\"  when require is used", 3
 	unless defined $qv_declare;
     my ($fh, $filename) = tempfile('tXXXXXXX', SUFFIX => '.pm', UNLINK => 1);
     (my $package = basename($filename)) =~ s/\.pm$//;
     print $fh <<"EOF";
 package $package;
-use base $CLASS;
+use base '$CLASS';
 1;
 EOF
     close $fh;
     # need to eliminate any other $qv_declare()'s
-    undef *{"main\::$qv_declare"};
-    ok(!defined(&{"main\::$qv_declare"}), "make sure we cleared $qv_declare() properly");
+    {
+        no strict 'refs';
+        undef *{"main\::$qv_declare"};
+    }
+    ok(!main->can($qv_declare), "make sure we cleared $qv_declare() properly");
     eval "use lib '.'; use $package qw/declare qv/;";
-    ok(defined(&{"main\::$qv_declare"}), "make sure we exported $qv_declare() properly");
-    isa_ok( &$qv_declare(1.2), $package);
+    die "Error from test: $@" if $@;
+    ok(main->can($qv_declare), "make sure we exported $qv_declare() properly");
```

Comment:
I don't think these need to be changed, and to be honest the old version is technically the more correct one.

Reply:
restored the previous version via 49481772